README.md

```diff
@@ -55,7 +59,7 @@ If you're building any app that will use .bim - let me know, I'll post it here :
 
 https://www.nuget.org/packages/dotbim/
 
-It may require importing another nuget for Newtonsoft.Json library: https://www.nuget.org/packages/Newtonsoft.Json/13.0.1
+It may require importing another nuget for Newtonsoft.Json library: https://www.nuget.org/packages/Newtonsoft.Json/9.0.1
```

Comment:
Hey Luiz :) we probably have to upgrade the Newtonsoft back to 13.0.1. I was not familiar with this issue before, but after merge I got an automatic message, that there were some major security issues in versions < 13.0.1, you can read more about it here: https://github.com/advisories/GHSA-5crp-9r3c-p9vr. Keeping it as it is makes the new users automatically download version with this issue, so if we can, we should prevent that action.

If I remember correctly downgrading was because of the fact that there could be some collision with existing Newtonsoft versions in different programmes that have < 13.0.1. versions already installed. Let me know if it's actually an issue.

Reply:
I'm aware of this warning, the only problem I see if I have an app that uses an old version like 11.0.1 I wouldn't be able to use this package if it requires a higher version.

It makes sense to use the version that is stable by default, or perhaps remove the package reference.
If the reference is removed, the nuget will not show the dependency, and the user will need to add the json package with whatever version he needs.

The problem is related to the high level of nesting, it seems like it will never happen in your code.

I really don't know what the best approach is, with the current approach the user can use a bigger version if needed.

I believe the current approach with 9.0.1 gonna work fine, the user can use a bigger version if needed in the main application.

Reply:
Ok, I see. What will happen if we will do other way around - the reference to the package by default would be 13.0.1 (which won't work for app that needs 11.0.1) and then you will downgrade it in your project? Is it possible this way, or there are some issues with it?
Or if such downgrading doesn't work with nugets - maybe using this library as a submodule, where you can locally downgrade to 11.0.1. This way user by default will have a safe version.
Generally it will be better as you said, to have safe as default one :) If both solutions above are problematic, then maybe you're right with removing the reference.